src/patronus/init.py

```diff
@@ -0,0 +1,68 @@
+from typing import Optional
+
+import httpx
+
+from . import config
+from .api import API, CTX_API
+from .evals.exporter import _CTX_EVAL_EXPORTER, BatchEvaluationExporter
+from .tracing.logger import _CTX_EVAL_LOGGER, _CTX_STD_LOGGER, create_logger, create_patronus_logger
+from .tracing.trace import _CTX_TRACER, create_tracer
+
+
+def init(
```

Comment:
To be sure: we won't allow to initiate multiple projects at once?

for example:
init("project_a")
init("project_b")

Reply:
Also i think you should add this to __init__.py so you can do
```
from patronus import init
```